nix/docker/default.nix

```diff
@@ -26,27 +26,6 @@ let
 
       config = {
         Cmd = [ "/bin/postgrest" "/etc/postgrest.conf" ];
-        Env = [
-          "PGRST_DB_URI=postgresql://?user=postgres"
-          "PGRST_DB_SCHEMA=public"
-          "PGRST_DB_ANON_ROLE="
-          "PGRST_DB_POOL=100"
```

Comment:
I found this gem. This seems to be a typo. According to some of @steve-chavez performance tests, this could have a negative impact on performance for our docker containers, right?

It's back to the default of `10` now.

Reply:
Woah! Nice catch!

Yes, too many connections will lead to contention: https://wiki.postgresql.org/wiki/Number_Of_Database_Connections#Summary

(Also confirmed that it's true, throughput decreased on my tests)